db-tests/src/test/java/gov/nasa/jpl/aerie/database/MerlinDatabaseTests.java

```diff
@@ -620,13 +622,23 @@ INSERT INTO plan_dataset (plan_id, dataset_id)
                                                          .formatted(newPlanId, planDatasetRecord.dataset_id())
                                                  );
       planDatasetInsertRes.next();
-      final var newOffsetFromPlanStart = Duration.parse(planDatasetInsertRes.getString("offset_from_plan_start"));
+      final var newOffsetFromPlanStart = new PGInterval(planDatasetInsertRes.getString("offset_from_plan_start"));
       planDatasetInsertRes.close();
 
-      final var calculatedOffset = offsetFromPlanStart.minus(Duration.ofMillis(newPlanStartTime.getTime()
-                                                                               - planStartTime.getTime()));
+      final var calculatedOffset = new PGInterval(offsetFromPlanStart.getValue());
+      calculatedOffset.setSeconds(calculatedOffset.getSeconds() - (newPlanStartTime.getTime() - planStartTime.getTime()) / 1000.0);
 
-      assertEquals(calculatedOffset, newOffsetFromPlanStart);
+      assertEquals(microsOfPGInterval(calculatedOffset), microsOfPGInterval(newOffsetFromPlanStart));
```

Comment:
I introduced it to try to get around this assertion error:
```
Expected :0 years 0 mons 0 days 0 hours 0 mins -3600.0 secs
Actual   :0 years 0 mons 0 days -1 hours 0 mins 0.0 secs
```

Reply:
I don't think we need to state that, since it's part of the `insertPlan` statement (either implicitly if we use the `insertPlan(int)` or explicitly as part of the `final var newPlanId` line.